tasty-query/shared/src/main/scala/tastyquery/TypeOps.scala

```diff
@@ -72,4 +72,65 @@ private[tastyquery] object TypeOps:
       super.expandBounds(tp)
     }
   }
+
+  // Tests around `matches`
+
+  /** The implementation for `tp1.matches(tp2)`. */
+  final def matchesType(tp1: Type, tp2: Type, relaxed: Boolean)(using Context): Boolean = tp1.widen match
```

Comment:
`relaxed` param is currently dead code - but this is a public API soo its not?

Reply:
`TypeOps` itself is `private[tastyquery]`, so this is not public.

I removed the `relaxed` parameter.